canonical/grafana-agent-k8s-operator
# FIX #42 - Downscaling Loki wipes its configuration

## Issue


See #42 

## Solution


Now we are no longer checking the event type, we are only checking if we have `loki_endpoints` provided by the Provider side of the relation

## Context



## Testing Instructions


1. Deploy grafana-agent
2. Deploy 3 loki units
3. Relate both charms `juju add-relation grafana-agent loki`
4. Down scale Loki `juju scale-application loki 2`
5. Verify that `loki` section in grafana-agent config file is NOT empty: `juju ssh --container agent agent/0 cat /etc/agent/agent.yaml`

## Release Notes
